db/data_migration/20160311093609_generate_content_ids_on_attachments.rb

```diff
@@ -0,0 +1,6 @@
+require 'securerandom'
+
+Attachment.where(content_id: nil).find_each do |attachment|
```

Comment:
How about just executing a single SQL update statement with NEWID() instead? This round-trips like heck otherwise.